composer.json

```diff
@@ -30,9 +30,9 @@
         "ext-pcre": "*",
         "ext-pdo": "*",
         "ext-zlib": "*",
-        "composer-runtime-api": "^2.0",
         "ausi/slug-generator": "^1.1",
         "bacon/bacon-qr-code": "^2.0",
+        "composer-runtime-api": "^2.0",
```

Comment:
Unrelated, please revert.

Reply:
Why was it in false order anyway?

Reply:
The order is correct.

1. PHP version
2. PHP extensions
3. Composer runtime API
4. Packages

See https://github.com/ergebnis/composer-normalize.

Reply:
This is the output produced by the `composer-json` helper.

Reply:
see https://github.com/contao/monorepo-tools/issues/18

Reply:
reverted in d5729bdee2c95a3c376b3d1be395e05b4831330d